dump/match_helper.py

```diff
@@ -1,6 +1,23 @@
 from dump.match_infra import MatchRequest
 from dump.helper import handle_multiple_keys_matched_error
 
+# Return dict helper methods
+
+def check_status(ret):
+    """ Check if the match request failed """
+    if ret["error"]:
+        return True, ret["error"]
+    else:
+        return False, ""
+
+def get_matched_keys(ret):
+    """ Return Matched Keys """
+    failed, err_str = check_status(ret)
+    if not failed:
+        return ret["keys"], err_str
+    else:
+        return [], ""
```

Comment:
Shouldn't this return error string when failed in case there is a use case to know the reason why it failed?

Reply:
you're corect, i'll update this